src/views/forms/FormTransferTransaction/FormTransferTransactionTs.ts

```diff
@@ -278,6 +273,30 @@ export class FormTransferTransactionTs extends FormTransactionBase {
         }
         this.formItems.recipient = !!this.recipient ? this.recipient : null;
 
+        this.formItems.messagePlain = this.message ? Formatters.hexToUtf8(this.message.payload) : '';
+        this.formItems.encryptMessage = false;
+        this.encyptedMessage = null;
+        // - maxFee must be absolute
+        this.formItems.maxFee = this.defaultFee;
+
+        // transaction details passed via router
+        this.importTransaction = this.$route.params.transaction || this.importedTransaction ? true : false;
+        this.resetMosaicsAndTriggerChange(this.importTransaction);
+        this.triggerChange();
```

Comment:
I thought you wanted to remove this line and call the `triggerChange` in `resetMosaicsAndTriggerChange` all the time (and remove the `triggerChange` method parameter) and that's why I proposed to change the method name to resetMosaics _AndTriggerChange_